git-github-introduction.md

```diff
@@ -0,0 +1,340 @@
+our projects are expected to be made using [pull requests](https://help.github.com/articles/using-pull-requests)
+to repositories of their GitHub organization.  For example, the Seattle repositories
+is hosted on the [SeattleTestbed](https://github.com/SeattleTestbed) organization.
+
+Git is a free and open source distributed [VCS](http://en.wikipedia.org/wiki/Version_control_system)
+(Version Control System).  A VCS helps manage changes made to files.  Users can run Git on their local
+computers and upload changes to remote repositories and files hosted on Github.  Contributors can edit files
+locally, upload their changes to Github, and then initiate a pull request. The following document
+covers the basics of using Git and submitting a pull request to one of our project's GitHub repository.
+
+
+
+
+
+## Git Installation
+Github provides platform-specific installers for Mac and Windows users.  Installation instructions
+are provided in the links below for each supported platform.
+
+[Mac](https://mac.github.com)
+
+[Windows](https://windows.github.com)
+
+[Linux](http://git-scm.com/download/linux)
+
+
+Linux users may install Git using their distribution's package manager.
+Installation commands for many Linux distributions are available at [Linux](http://git-scm.com/download/linux).
+
+For example, Debian-based Linux distributions can use [APT](http://en.wikipedia.org/wiki/Advanced_Packaging_Tool) (Advanced Packaging Tool).
+```
+$ sudo apt-get install git
+```
+
+
+## Configure Git User Information
+
+After installing Git, the next step is for the user to configure his/her author information (username and email address) so that Git will correctly log the author of a commit. 
+
+
+To configure Git author information, open the Terminal and type **git config --global user.name "user's name"**, which will allow you to configure Git on your local machine. Each repository you create will use the same user (author) configuration if you use the global flag (as in the provided example). 
+
+```
+$ git config --global user.name "John Smith"
+
+$ git config --global user.email "john.smith@example.com"
+```
+
+You can check your user information by typing the following command:
+
+```
+$ git config --list
+```
+
+
+## Configure Github User Account
+Before a pull request can be submitted to a [SeattleTestbed](https://github.com/SeattleTestbed)
+repository, a user account (free) must be created on Github.  This user account is also needed to create
+new repositories and fork ones that already exist on Github.
+
+
+A Github user account can be created at https://github.com/join
+
+Once users create their accounts, user-specific home pages are created. For example: ''https://github.com/john-smith''.
+A user-specific home page can contain repositories that the user has created or forked, contribution activity,
+and organizations he/she has joined. 
+
+
+## Fork a `SeattleTestbed` Repository
```

Comment:
Okay, you explained offline that we shouldn't specifically use `SeattleTestbed` in the subsection title.  `Fork a Repository` should be a better title.